readme.md

```diff
@@ -31,6 +31,7 @@ const configuration = {
   appDescription: null, // Your application's description. `string`
   developerName: null, // Your (or your developer's) name. `string`
   developerURL: null, // Your (or your developer's) URL. `string`
+  cacheBustingQueryParam: null, // Query parameter added to all URLs that acts as a cache busting system. `string`
```

Comment:
This field is defined as `string | undefined`. `null` does not belong to that type.

Reply:
Yeah. Old properties are not very strict.